Startup.cs

```diff
@@ -39,18 +38,24 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
             else
             {
                 app.UseExceptionHandler("/Error");
+                // The default HSTS value is 30 days. You may want to change this for production scenarios, see https://aka.ms/aspnetcore-hsts.
                 app.UseHsts();
             }
 
             app.UseHttpsRedirection();

```

Comment:
Https redirection should be forced only in prod mode (or removed altogether). Otherwise the ugly `Warning: Potential Security Risk Ahead` prompt pops up on first load.

Reply:
There's an `env.IsDevelopment()` check above it, am I missing something?

Reply:
![image](https://user-images.githubusercontent.com/18740853/75518769-5aeee580-5a0a-11ea-9453-37874908bf3e.png)

The `app.UseHttpsRedirection();` is executed in both environments. And as a result...

![image](https://user-images.githubusercontent.com/18740853/75518850-938ebf00-5a0a-11ea-9607-2a9a909691b3.png)

Not a real problem, as clients can hit `advanced > proceed`, but could be avoided altogether.